src/components/StudyCard/StudyCard.tsx

```diff
@@ -0,0 +1,86 @@
+import { Link } from 'react-router-dom';
+import moment from 'moment';
+import PhotoIcon from '@mui/icons-material/Photo';
+import { MbtiTag } from '@components';
```

Comment:
```suggestion
import MbtiTag from '@components/MbtiTag/MbtiTag';
```

위와 같이 변경하면 키아님이 질문하신 것처럼 `import StudyCard from './StudyCard'` 형태로 스토리 파일에서 호출이 가능하나, 이러면 `components/index.ts`를 사용하는 이유가 없어지기 때문에 그냥 지금 방식대로 해야할 것 같습니다..😥